Makefile

```diff
@@ -1,6 +1,11 @@
 all: build
 .PHONY: all
 
+# Since k8s 1.24 this is required to enable GCP PD tests.
+# Here it's mostly used by update targets. See:
+# https://github.com/kubernetes/kubernetes/pull/109541
+export ENABLE_STORAGE_GCE_PD_DRIVER := yes
```

Comment:
This doesn't look like the right place for setting this env var, please advise.

Reply:
CC @bparees 
Any ideas?

Reply:
you can set it here, to put it a bit closer to where it matters:

https://github.com/openshift/origin/blob/6438b04e175f9e2ce434b7deb4b786f602eb886b/hack/update-generated.sh#L6

e.g.:
```
ENABLE_STORAGE_GCE_PD_DRIVER=yes go generate -mod vendor ./test/extended
```
should do it.

the bigger problem is you also need to set this in the actual CI jobs where you want this test to run.